### PR TITLE
Remove few output deprecations

### LINKED ISSUE #549: Remove deprecated JSON keys from output
Remove the following keys from the JSON output: https://github.com/infracost/infracost/blob/master/internal/output/output.go#L18-L20

The HTML output currently relies on these so this will need to be done with https://github.com/infracost/infracost/issues/548.

### PR DESCRIPTION
**Pull request by sinabakh**
Fix #549 

Changes:
* Remove deprecated fields
* Remove deprecated table format

### CODE REVIEW

File: cmd/infracost/run.go [hunk 1]
```diff
@@ -154,9 +154,6 @@ func runMain(cmd *cobra.Command, cfg *config.Config) error {
 	case "diff":
 		b, err = output.ToDiff(r, opts)
 		out = fmt.Sprintf("\n%s", string(b))
-	case "table_deprecated":
```
Comment (aliscott) [inline cmd/infracost/run.go:157]: We will need to remove the [root_deprecated](https://github.com/infracost/infracost/blob/master/cmd/infracost/root_deprecated.go) command as well since it uses this format. Happy for that to be done in a separate PR though. What do you think?

**Review by aliscott — APPROVED** [review]
LGTM thanks @sinabakh!

**Comment by sinabakh** [comment]
@aliscott I removed few more deprecations as you mentioned. Since it initially contained more changes than the original issue I pushed the codes on this branch.
However, the commits are atomic and can be merged without a squash.

**Review by aliscott — APPROVED** [review]
This is awesome! Thanks @sinabakh
